soccer/src/soccer/planning/planner/collect_path_planner.cpp

```diff
@@ -453,6 +715,8 @@ bool CollectPathPlanner::is_done() const {
     // (see process_state_transition())
     if (current_state_ != Control) {
```

Comment:
return current_state_ == Control?

Also, if we're not using the below code do we need Control?